src/StyleSheet.js

```diff
@@ -128,40 +126,40 @@ export default class StyleSheet {
   }
 
   /**
-   * Create rules, will render also after stylesheet was rendered the first time.
+   * Create styles, will render also after stylesheet was rendered the first time.
```

Comment:
hmm actually in this case, its create rules, we take styles and create rules))))